lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+        has_comments_or_approvals = (has_comments? or approvals.any?)
+
+        case self.class.minimum_approvals
+        when "0"
+          true
+        when "", nil
+          # Approvals from every commenter
+          has_comments_or_approvals && reviewers_pending_response.empty?
+        else
+          approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
```

Comment:
Sorry that example wasn't the clearest.  Ultimately with Bitbucket we don't consider the content of the comments at all.  They have a special "Approval" api that we take advantage of that is built into their interface to skirt the need for approval comments.  When we've had to work on Bitbucket before, we'd find ourselves "Approving" pull requests using their API and then having to go back and add `LGTM` comment because reflow would complain.  We'd like to keep that in tact :wink:

So for the Github PR we'll need to either override the method entirely, or figure out a special combination of override + call to `super` to fallback on.